jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -527,7 +528,8 @@ pipeline {
                                                     parameters: [
                                                         string(name: 'TEST_MANIFEST', value: TEST_MANIFEST),
                                                         string(name: 'BUILD_MANIFEST_URL', value: buildManifestUrl),
-                                                        string(name: 'BUILD_MANIFEST_URL_OPENSEARCH', value: buildManifestUrlOpenSearch)
+                                                        string(name: 'BUILD_MANIFEST_URL_OPENSEARCH', value: buildManifestUrlOpenSearch),
+                                                        booleanParam(name: 'UPDATE_GITHUB_ISSUES', value: true)
```

Comment:
Default it to false maybe?

Reply:
This is the distribution build job - we want it to keep sending notifications, no?  Are any 'unsupported' jobs run with this task?

Reply:
Sorry missed the fact that this is a distribution build job. I think we might re-think the logic once https://github.com/opensearch-project/opensearch-build/issues/4301 feature support is added.